apptools/apptools-android-tests/apptools/CI/crosswalk_pkg_basic.py

```diff
@@ -282,26 +319,30 @@ def test_keep_project(self):
                     apkLength = apkLength + 1
                     appVersion = apks[i].split('-')[1]
             self.assertEquals(apkLength, 1)
-        projectDir = output[0].split(" * " + os.linesep)[-1].split(' ')[-1].strip(os.linesep)
+        project_dir = None
+        for line in output.split('\n'):
+            match = re.search(r'Keeping build tree in (.+)$', line)
+            if match is not None:
+                project_dir = match.group(1)
         comm.run(self)
         comm.clear("org.xwalk.test")
-        self.assertEquals(return_code, 0)
-        self.assertIn("app", os.listdir(projectDir))
-        self.assertIn("prj", os.listdir(projectDir))
+        self.assertTrue(os.path.isdir(project_dir))
+        self.assertTrue(os.path.isdir(os.path.join(project_dir, "app")))
+        self.assertTrue(os.path.isdir(os.path.join(project_dir, "prj")))
 
     def test_target_arch(self):
         comm.setUp()
         os.chdir(comm.XwalkPath)
         comm.clear("org.xwalk.test")
         os.mkdir("org.xwalk.test")
         os.chdir('org.xwalk.test')
-        if comm.BIT == "64":
-            cmd = comm.HOST_PREFIX + comm.PackTools + \
-                "crosswalk-pkg --platforms=android --android=" + comm.ANDROID_MODE + " --crosswalk=" + comm.crosswalkzip + ' --targets="arm64-v8a x86_64" ' + comm.ConstPath + "/../testapp/create_package_basic/"
-        else:
-            cmd = comm.HOST_PREFIX + comm.PackTools + \
-                "crosswalk-pkg --platforms=android --android=" + comm.ANDROID_MODE + " --crosswalk=" + comm.crosswalkzip + ' --targets="armeabi-v7a x86" ' + comm.ConstPath + "/../testapp/create_package_basic/"
-        return_code = os.system(cmd)
+        self.invokeCrosswalkPkg([
+            '--platforms=android',
+            '--android=' + comm.ANDROID_MODE,
+            '--crosswalk=' + comm.crosswalkzip,
+            '--targets=' + comm.BIT,
+            comm.ConstPath + '/../testapp/create_package_basic/',
+        ])
```

Comment:
def test_target_arch(self) checkpoint: pack apk with --targets="arm64-v8a x86_64" or --targets="armeabi-v7a x86". ”--targets=64“ and ”--targets=32“ have been covered in other tests